src/Microsoft.OData.Edm/Vocabularies/Annotations/EdmVocabularyAnnotation.cs

```diff
@@ -80,9 +80,8 @@ public IEdmExpression Value
         }
 
         /// <summary>
-        /// Gets whether the annotation uses a default value.
-        /// In vNext, need refactor this.
+        /// Checks if the annotation uses a default value. </param>
         /// </summary>
-        internal bool UseDefault { get; set; }
+        public bool UseDefault { get; set; }

```

Comment:
Why are we making these public ?

Reply:
We added this to a public interface that this class implements.